src/events/cd-user-ticket-list-item.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <div class="cd-user-ticket-list-item">
+    <div class="cd-user-ticket-list-item__details">
+      <header class="cd-user-ticket-list-item__header">
+        <h3 class="cd-user-ticket-list-item__name">
+          {{ event.name }}
+        </h3>
+      </header>
+      <div class="cd-user-ticket-list-item__datetime">
+        <div v-if="!isRecurring" class="cd-user-ticket-list-item__date-timestamp">
+          {{ event.dates[0].startTime | cdDateFormatter }}
+        </div>
+        <div v-else>
+          <div class="cd-user-ticket-list-item__date-series">{{ $t('Next in series:') }}</div>
+          <span class="cd-user-ticket-list-item__date-timestamp">{{ nextStartTime |  cdDateFormatter }}</span>
+        </div>
+        <div class="cd-user-ticket-list-item__times-timestamp">
+          {{ event.dates[0].startTime | cdTimeFormatter }} - {{ event.dates[0].endTime | cdTimeFormatter }}
```

Comment:
Might as well use `formattedStartTime` and `formattedEndTime` here instead. IIRC filters are gonna be deprecated, so all the better really.

Reply:
And yes, I am aware that you probably just copy and pasted cd-event-list-item and it had this already, but still 😛